src/components/Tips/Tips.tsx

```diff
@@ -0,0 +1,19 @@
+import { FC, Fragment } from 'react'
+import { useSelector } from 'react-redux'
+import TipId from '../../@types/TipId'
+import getTip from '../../selectors/getTip'
+import NewSubthoughtTip from './NewSubthoughtTip'
+import NewThoughtTip from './NewThoughtTip'
+
+/** A container for rendering all of the tips. */
+const Tips: FC = () => {
+  const tip = useSelector(getTip)
+  return (
+    <Fragment>
```

Comment:
Prefer the shorthand: `<>` and `</>`

Reply:
Switched: 5e930a8e8767ad52282fc4d3c5911e11db5e4448